meta-mbl-apps/recipes-support/swupdate/swupdate-headers_2019.11.bb

```diff
@@ -12,9 +12,7 @@ SUMMARY = "Header files for building handlers for swupdate"
 DESCRIPTION = "${SUMMARY}"
 FILESEXTRAPATHS_prepend = "${OEROOT}/layers/meta-swupdate/recipes-support/swupdate/swupdate:"
 
-PACKAGES = "${PN}-dev"
-FILES_${PN}-dev = "${includedir}/*"
```

Comment:
I don't think we should remove these lines: `swupdate_${PV}.bb` adds a bunch of extra packages that we don't want, and might set `FILES_${PN}-dev` to something other than what we want here.

I wonder if the error you saw was because the default value of `RDEPENDS_${PN}-dev` contains `${PN}`... I suggest that instead of deleting these lines and adding `ALLOW_EMPTY_${PN}`, we try setting `RDEPENDS_${PN}-dev = ""`

I'll try that out...

Reply:
https://github.com/ARMmbed/meta-mbl/pull/811/commits/010e1522473e8bf622e86befc9ef3004a157a875